polaris-react/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.module.scss

```diff
@@ -50,31 +74,30 @@
     $layout-width-inner-spacing-base;
   height: 100%;
   margin: 0 auto;
-  padding: 0 var(--p-space-400);
-
-  @media (--p-breakpoints-sm-up) {
-    padding: 0 var(--p-space-500);
-  }
-
-  @media (--p-breakpoints-md-up) {
-    padding: 0 var(--p-space-800);
-  }
+  padding: var(--p-space-100);
+  padding-left: var(--p-space-200);
+  z-index: 1;
 }
 
 .fullWidth {
-  max-width: none;
-  padding: 0 var(--p-space-400);
+  // max-width: none;
+  // padding: 0 var(--p-space-400);
```

Comment:
What should we do with this `fullWidth` [prop](https://coverage.polaris.shopify.io/usage/element?repo=shopify%2Fweb&type=components&element=%3CContextualSaveBar%3E&props=discardAction%2CfullWidth%2Cmessage%2CsaveAction)?